content/en/continuous_integration/setup_pipelines/github.md

```diff
@@ -50,5 +75,8 @@ The [Pipelines][4] and [Pipeline Executions][5] pages populate with data after t
 [1]: https://docs.github.com/actions
 [2]: https://docs.github.com/developers/apps/getting-started-with-apps/about-apps
 [3]: https://app.datadoghq.com/account/settings#integrations/github-apps
-[4]: https://app.datadoghq.com/ci/pipelines
-[5]: https://app.datadoghq.com/ci/pipeline-executions
+[4]: https://app.datadoghq.com/ci/setup/pipeline?provider=github
```

Comment:
I wonder whether we should link to the onboarding or the new settings page

Reply:
Yes, I wasn't sure about that, as it was the initial setup I went to the onboarding but I can change it to the settings page if you think it is better